### PR TITLE
Run over old systemd platform

### PR DESCRIPTION
**Pull request by JulienBalestra**
### What does this PR do?

This PR allows to run pupernetes over old systemd platform like ubuntu 14.04.
As it's the only choice in several SaaS CI (circleci, travis) I managed to make it works over this distribution.

### Motivation

Be able to run pupernetes in circleci. 

This is very useful to be able to test pupernetes in its own CI system.
But, it's also very interesting for others projects like [datadog-agent](https://github.com/DataDog/datadog-agent) :tada:

### CODE REVIEW

**Comment by JulienBalestra** [comment]
There are some issues with the kubelet on Kubernetes 1.5 and 1.6. 
I propose to postpone them for later to reduce the scope of this PR.

File: pkg/setup/templates/1.10.go [hunk 1]
```diff
@@ -21,7 +21,6 @@ ExecStart={{.RootABSPath}}/bin/hyperkube kubelet \
 	--healthz-port=10248 \
 	--cert-dir=/var/lib/p8s-kubelet/pki \
 	--kubeconfig={{.RootABSPath}}/manifest-config/kubeconfig-insecure.yaml \
-	--cloud-provider="" \
```
Comment (CharlyF) [inline pkg/setup/templates/1.10.go]: Are we planning on supporting this flag at some point ? (referring to @kindermoumoute's issue)

Comment (JulienBalestra) [reply pkg/setup/templates/1.10.go]: Yes we should, but it's not something needed for our use case right now.

**Comment by JulienBalestra** [comment]
@CharlyF I'll submit a PR to include `golint` :+1:

**Pull request merged**